### PR TITLE
Refactor/footer stories

### PR DESCRIPTION
**Pull request by dannyk3941**
#### What's this PR do?
- Create footers for the CBP Design System
- Add `cbp-font-style-italic` utility class
- Change viewport sizes for Storybook

#### Where should the reviewer start?
- `npm run styles-build`
- `npm run styles-sb` -> Patterns -> Footer

#### How should this be manually tested?
- Resize viewport for footer stories.

### CODE REVIEW

File: packages/styles/src/sass/components/footer/_external.scss [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  }
+}
+
+.cbp-external-footer__links {
+  font-size: var(--cbp-font-size-heading-xs);
+  font-weight: var(--cbp-font-weight-medium);
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
```
Comment (dgibson666) [inline packages/styles/src/sass/components/footer/_external.scss:55]: I think if you set columns to 2 or 3 and column-width: 8rem that you get a good balance without media queries. Then you can delete these properties from the larger media queries and let them inherit from here.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns/Basic_Concepts_of_Multicol#specifying_both_number_and_width_of_columns

File: packages/styles/src/sass/components/footer/_internal.scss [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  & > i {
+    margin-right: var(--cbp-space-1x);
+  }
+
+  &:hover {
+    color: var(--cbp-color-interactive-primary-base);
+  }
+
+  &:focus {
+    color: var(--cbp-color-interactive-focus-light) !important;
```
Comment (dgibson666) [inline packages/styles/src/sass/components/footer/_internal.scss:16]: What is overriding this that you need `!important`? Any place you have `!important`, see if you can get rid of it either with ordering of the rules (cascade) or specificity.

**Review by dgibson666 — CHANGES_REQUESTED** [review]
Looks good overall. See code-level comments.

File: packages/styles/src/sass/components/footer/_internal.scss (hunk 2 above)
Comment (dannyk3941) [reply packages/styles/src/sass/components/footer/_internal.scss]: re-ordered pseudo classes in the mixin, will be taken out in the next buttons PR and imported over.

File: packages/styles/src/components/footer/footer.stories.js [hunk 3]
```diff
@@ -1,40 +1,97 @@
...
+        </section>
+        <div class="cbp-footer__security">
           <div>
-            <h6>Infosec Classification: FOUO</h6>
-            <p>This page contains information, in whole or part, that is marked FOR OFFFICIAL USE ONLY (FOUO). </p>
-            <a href="#"><i class="fas fa-external-link-alt"></i>&nbsp;Learn More About InfoSec</a>    
+            <img src="../assets/images/cbp-icon/SECURITY ICON - LES FOUO.svg" alt="security icon">
+            <div>
+              <h6 class="cbp-footer__heading">Infosec Classification: FOUO</h6>
+              <p class="cbp-footer__text">This page contains information, in whole or part, that is marked FOR OFFFICIAL USE ONLY (FOUO). <a href="#"><i class="fas fa-external-link-alt"></i>Learn More About InfoSec</a></p>
```
[the remark this answers is not in the document]
Comment (dannyk3941) [reply packages/styles/src/components/footer/footer.stories.js]: Added "external link" to title attribute for these links.

File: packages/styles/src/components/footer/footer.stories.js [hunk 4]
```diff
@@ -1,40 +1,97 @@
...
+        </ul>
+      </nav>
       <div class="cbp-footer__info">
-        <div>
-          <h6><i class="fas fa-headset"></i> Application Support</h6>
-          <p class="cbp-footer__supplement">This application is maintained by The Office of Information Technology: TASPD.</p>
-          <ul class="cbp-contact-list">
+        <section class="cbp-footer__support">
+          <h6 class="cbp-footer__heading"><i class="fas fa-headset"></i> Application Support</h6>
+          <p class="cbp-footer__text">This application is maintained by The Office of Information Technology: TASPD.</p>
```
[the remark this answers is not in the document]
Comment (dannyk3941) [reply packages/styles/src/components/footer/footer.stories.js]: Used `<abbr>` with title attribute.

**Review by dgibson666 — APPROVED** [review]
Looks good.

**Pull request merged**